worldedit-core/src/main/java/com/fastasyncworldedit/core/util/TextureUtil.java

```diff
@@ -810,7 +810,7 @@ public void loadModTextures() throws IOException {
                 new File(Fawe.platform().getDirectory() + "/" + Settings.settings().PATHS.TEXTURES + "/")
                         .mkdirs();
                 try (BufferedInputStream in = new BufferedInputStream(
-                        new URL("https://piston-data.mojang.com/v1/objects/c0898ec7c6a5a2eaa317770203a1554260699994/client.jar")
+                        new URL("https://piston-data.mojang.com/v1/objects/055b30d860ead928cba3849ba920c88b6950b654/client.jar")
```

Comment:
We already downloaded the client jar at this point if I see this right, couldn't we therefore get rid of this somehow?

Reply:
Gonna rebase the pr, yes - Alex cherry picked the client jar download from a commit I've made in this PR